homeassistant/components/zabbix/__init__.py

```diff
@@ -169,41 +172,47 @@ class ZabbixThread(threading.Thread):
 
     MAX_TRIES = 3
 
-    def __init__(self, hass, zabbix_sender, event_to_metrics):
+    def __init__(
+        self,
+        zabbix_sender: ZabbixSender,
+        event_to_metrics: Callable[
+            [Event, set[str], set[str]], list[ZabbixMetric] | None
+        ],
+    ) -> None:
         """Initialize the listener."""
         threading.Thread.__init__(self, name="Zabbix")
-        self.queue = queue.Queue()
+        self.queue: queue.Queue = queue.Queue()
```

Comment:
Isn't this inferred, this can be omitted?

Reply:
mypy didn't like it iirc